build_runner/lib/src/asset/cache.dart

```diff
@@ -36,6 +38,9 @@ class CachingAssetReader implements AssetReader {
       _canReadCache.putIfAbsent(id, () => _delegate.canRead(id));
 
   @override
+  Future<Digest> digest(id) => _delegate.digest(id);
```

Comment:
Note that I removed caching of digests from here - they are cached on `AssetNode`s already so this would be duplicative.